python/requirements.txt

```diff
@@ -1,4 +1,5 @@
-construct==2.5.2
+construct==2.8.12
```

Comment:
We may need a check in the library somewhere that asserts if this version is incorrect. People don't always update their dependencies.